app/src/main/java/com/pokemonshowdown/app/BattleFragment.java

```diff
@@ -1755,7 +1755,8 @@ private AlertDialog parseMoveTargetDialog(final JSONArray active, final int move
         final int numFoes = foeIndex;
         final int currentActive = mCurrentActivePokemon;
         switch (target) {
-            case "normal":
+            case "any": //can hit anything on the BG
```

Comment:
Oops seems like i forgot to commit something lol
On 29 Mar 2015 23:05, "Nam Thai" <notifications@github.com> wrote:

> In app/src/main/java/com/pokemonshowdown/app/BattleFragment.java
> <https://github.com/NamTThai/PokemonShowdownAndroidClient/pull/83#discussion_r27358985>
> :
>
> > @@ -1755,7 +1755,8 @@ private AlertDialog parseMoveTargetDialog(final JSONArray active, final int move
> >          final int numFoes = foeIndex;
> >          final int currentActive = mCurrentActivePokemon;
> >          switch (target) {
> > -            case "normal":
> > +            case "any": //can hit anything on the BG
>
> Are you implying that any and normal are handled the same way? Doesn't
> sound like that to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NamTThai/PokemonShowdownAndroidClient/pull/83/files#r27358985>
> .
>